game_eggs/steamcmd_servers/astro_colony/egg-astro-colony.json

```diff
@@ -4,50 +4,38 @@
\n\r\n## install game using steamcmd\r\n.\/steamcmd.sh +force_install_dir \/mnt\/server +login ${STEAM_USER} ${STEAM_PASS} ${STEAM_AUTH} $( [[ \"${WINDOWS_INSTALL}\" == \"1\" ]] && printf %s '+@sSteamCmdForcePlatformType windows' ) +app_update ${SRCDS_APPID} $( [[ -z ${SRCDS_BETAID} ]] || printf %s \"-beta ${SRCDS_BETAID}\" ) $( [[ -z ${SRCDS_BETAPASS} ]] || printf %s \"-betapassword ${SRCDS_BETAPASS}\" ) ${INSTALL_FLAGS} validate +quit ## other flags may be needed depending on install. looking at you cs 1.6\r\n\r\n## set up 32 bit libraries\r\nmkdir -p \/mnt\/server\/.steam\/sdk32\r\ncp -v linux32\/steamclient.so ..\/.steam\/sdk32\/steamclient.so\r\n\r\n## set up 64 bit libraries\r\nmkdir -p \/mnt\/server\/.steam\/sdk64\r\ncp -v linux64\/steamclient.so ..\/.steam\/sdk64\/steamclient.so\r\n\r\n## add below your custom commands if needed\r\n\r\n# We need to get the server download url from the official Linux installation script until it's released on steam.\r\n# For now there is only windows executable.\r\n# Steam APP id: 2662210\r\n# https:\/\/raw.githubusercontent.com\/FuriosIra\/AstroColonyServerScript\/main\/InstallAstroColonyLinuxServer.sh\r\nif [[ \"${AUTO_UPDATE}\" == 1 || ! -f \"AstroColonyLinuxServer.zip\" ]]; then\r\n    cd \/mnt\/server\r\n    wget -q --no-check-certificate -O \"InstallAstroColonyLinuxServer.sh\" \"https:\/\/raw.githubusercontent.com\/FuriosIra\/AstroColonyServerScript\/main\/InstallAstroColonyLinuxServer.sh\"\r\n    serverFoldersLink=$(grep -o 'serverFoldersLink=\"[^\"]*' \"InstallAstroColonyLinuxServer.sh\" | sed 's\/serverFoldersLink=\"\/\/')\r\n    eval \"serverFoldersLink=\\\"$serverFoldersLink\\\"\"\r\n    wget --load-cookies \/tmp\/cookies.txt \"$serverFoldersLink\" -O AstroColonyLinuxServer.zip && rm -rf \/tmp\/cookies.txt\r\n    unzip -o AstroColonyLinuxServer.zip\r\n    chmod +x LinuxServer\/AstroColonyServer.sh\r\n    # add steamclient.so to Astro Colony otherwise it wont display on serverlist\r\n    cp -v linux64\/steamclient.so LinuxServer\/AstroColony\/Binaries\/Linux\/steamclient.so\r\nfi\r\n\r\n## set up settings\r\ncreate_server_settings_ini() {\r\n    cat > ServerSettings.ini <<EOF\r\n[\/Script\/AstroColony.EHServerSubsystem]\r\nServerPassword=${SERVER_PASSWORD}\r\nSeed=${SERVER_MAPSEED}\r\nMapName=${SERVER_MAPNAME}\r\nMaxPlayers=${SERVER_MAXPLAYERS}\r\nShouldLoadLatestSavegame=${SERVER_LOADLATESTSAVEGAME}\r\nAdminList=${SERVER_ADMINS} \r\nSharedTechnologies=${SERVER_SHAREDTECHNOLOGIES}\r\nOxygenConsumption=${SERVER_OXYGENCONSUMPTION}\r\nFreeConstruction=${SERVER_FREECONSTRUCTION}\r\nAutosaveInterval=${SERVER_AUTOSAVEINTERVAL}\r\nAutosavesCount=${SERVER_AUTOSAVESCOUNT}\r\nEOF\r\n}\r\n\r\nmkdir -p LinuxServer\/AstroColony\/Saved\/Config\/LinuxServer\r\ncd LinuxServer\/AstroColony\/Saved\/Config\/LinuxServer\r\ncreate_server_settings_ini\r\n\r\n## install end\r\necho \"-----------------------------------------\"\r\necho \"Installation completed...\"\r\necho \"-----------------------------------------\"",
```

Comment:
sry, but you cant use that script to install the server

`https:\/\/raw.githubusercontent.com\/FuriosIra\/AstroColonyServerScript\/main\/InstallAstroColonyLinuxServer.sh`

Did you ever read, how this script works?